docs/_posts/Ahmetemintek/2023-05-31-sbertresolve_icd10cm_augmented_billable_hcc_en.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: model
+title: Sentence Entity Resolver for Billable ICD10-CM HCC Codes (sbertresolve_icd10cm_augmented_billable_hcc)
+author: John Snow Labs
+name: sbertresolve_icd10cm_augmented_billable_hcc
+date: 2023-05-31
+tags: [en, clinical, entity_resolution, icd10cm, billable, hcc, licensed]
+task: Entity Resolution
+language: en
+edition: Healthcare NLP 4.4.2
+spark_version: 3.0
+supported: true
+annotator: SentenceEntityResolverModel
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This model maps clinical entities and concepts to ICD-10-CM codes using `sbert_jsl_medium_uncased` sentence bert embeddings and it supports 7-digit codes with HCC status. It also returns the official resolution text within the brackets inside the metadata. The model is augmented with synonyms, and previous augmentations are flexed according to cosine distances to unnormalized terms (ground truths).
```

Comment:
lets share Hierarchical Condition Categories (HCC) when we mention for the first time.

Reply:
Add: In the result, look for the `all_k_aux_labels` parameter in the metadata to get HCC status. This column can be divided to get further details: billable status || hcc status || hcc score. For example, if `all_k_aux_labels` is like `1||1||19` which means the `billable status` is 1, `hcc status` is 1, and h`cc score` is 19.